src/Exception/MediaTypeDoesNotExistException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Drupal\media_mpx\Exception;
+
+/**
+ * Class MediaTypeDoesNotExistException.
+ *
+ * @package Drupal\media_mpx\Exception
+ */
+class MediaTypeDoesNotExistException extends \Exception {}
```

Comment:
Should this extend `\Drupal\Core\Entity\EntityStorageException`? Or if not, perhaps `\RuntimeException`?

Reply:
Changed 👍